helpdesk-message-proxy-service/src/main/java/org/symphonyoss/symphony/bots/helpdesk/messageproxy/ChatListener.java

```diff
@@ -33,7 +36,7 @@ public ChatListener(TicketManagerService ticketManagerService, HelpDeskAi helpDe
 
   @Override
   public void onMessage(SymMessage symMessage) {
```

Comment:
Please fix the unit tests for this method.

Reply:
done